### PR TITLE
wip: working example for multithreading

### PR DESCRIPTION
**Pull request by jaswinder6991**

### CODE REVIEW

**Comment by jaswinder6991** [comment]
Fixes #47

**Review by ckshitij — APPROVED** [review]

**Review by ckshitij — CHANGES_REQUESTED** [review]
Please check and update all the **Arc< dyn Provider >** with **( Arc< dyn Provider + Send + Sync>)** below are function names: 

- view_function 
- view_state
- get_access_key
- state

File: near-accounts/src/accounts.rs [hunk 1]
```diff
@@ -26,7 +26,7 @@ use std::sync::Arc;
 #[derive(Clone)]
 pub struct TransactionSender {
     pub signed_transaction: SignedTransaction,
-    provider: Arc<dyn Provider>,
+    provider: Arc<dyn Provider + Send + Sync>,
```
Comment (ckshitij) [inline near-accounts/src/accounts.rs:29]: [Suggestion]:  you can define Alias for the same as well, ```type ProviderArc = Arc<dyn Provider + Send + Sync>;```

**Review by ckshitij — APPROVED** [review]
One minor comment, apart from this changes looks good to me.

**Pull request merged**